avm/res/document-db/mongo-cluster/tests/e2e/waf-aligned/main.test.bicep

```
@@ -46,9 +46,15 @@ module testDeployment '../../../main.bicep' = [
     name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
     params: {
       name: '${namePrefix}${serviceShort}001'
+      tags: {
+        environment: 'dev'
+        role: 'validation'
+        type: 'waf-aligned'
+      }
       administratorLogin: 'Admin001'
       administratorLoginPassword: password
-      nodeCount: 2
+      highAvailabilityMode: 'SameZone'
```

```suggestion
      highAvailabilityMode: 'ZoneRedundant'
```
Might be better for WAF-alignment.